docs/quickstart.md

```diff
@@ -405,7 +405,7 @@ Try one of the following resources to continue your Quix learning journey:
 
 * [Quix Streams GitHub](https://github.com/quixio/quix-streams){target=_blank}
 
-* [Quix definitions](../platform/definitions.html)
+* [Quix Platform glossary](../platform/glossary.md)
```

Comment:
This link will only work on quix.io/docs, it is probably worth replacing all these relative links with an absolute link.

Reply:
Good point!

Reply:
OK now @peter-quix ?